pages/common/yesod.md

```diff
@@ -0,0 +1,19 @@
+# yesod
+
+> Yesod web framework CLI helper.
+
+- Install Yesod CLI tool in a Yesod scaffolding:
+
+`stack build yesod-bin cabal-install --install-ghc`
+
+- Start development server:
+
+`stack -- exec yesod devel`
```

Comment:
I am a bit confused here. You mention the tool as `yesod`, but the command you use is `stack`. So should it be stack ?